deploy/helm-chart/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: promscale
-version: 0.7.2
+version: 0.7.3
```

Comment:
I think we bump this during the release process, not during PRs

Reply:
I bumped it as helm linter is complaining when we don't bump the version (as seen inhttps://github.com/timescale/promscale/actions/runs/1677448496). We basically have two choices of how we can proceed:
1) disable linter check `check-version-increment` and release helm chart only when new promscale is released
2) leave linter check and release helm chart as soon as there is any change (using https://github.com/helm/chart-releaser-action).

In both cases, chart can be published automatically to s3. The benefit of the first solution is that we don't spam promscale release channels (as seen for example in https://github.com/kubernetes/ingress-nginx/releases). With the latter approach, we can benefit from faster feature availability (this would come in handy for promscale on cloud).

Regardless, I have disabled linter check for now as the decision doesn't need to be done for this PR. However, we need to have a path forward before approving https://github.com/timescale/promscale/pull/1030.

Reply:
I think spamming the release channel is not very user friendly for people subscribed to releases. I'd rather keep releases synced for now. If we want something for cloud we could potentially do something custom